tools/metricsdocs/metricsdocs.go

```diff
@@ -3,6 +3,8 @@ package main
 import (
 	"fmt"
 
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/monitoring/rules"
```

Comment:
this should be grouped with `"github.com/kubevirt/hyperconverged-cluster-operator/pkg/monitoring/metrics"`

https://github.com/unknwon/go-code-convention/blob/main/en-US.md#import-packages

Reply:
fixed